midend/local_copyprop.h

```diff
@@ -131,6 +131,20 @@ class LocalCopyPropagation : public PassManager {
         if (!typeChecking)
             typeChecking = new TypeChecking(refMap, typeMap, true);
         passes.push_back(typeChecking);
+        policy =
```

Comment:
Please move this to a separate psa backend pass.

Reply:
This will break any backend that tries to use LocalCopyProp with a different policy.  This should be moved to backends/bmv2/psa_switch/midend.cpp, and passed as an argument to the LocalCopyProp constructor...

Reply:
Done moving to backends/bmv2/psa_switch/midend.cpp